src/Filesystem/Asset.php

```diff
@@ -1,54 +1,42 @@
 <?php
 
-namespace Kirby\Cms;
-
-use Kirby\Toolkit\Properties;
+namespace Kirby\Filesystem;
 
 /**
  * Anything in your public path can be converted
  * to an Asset object to use the same handy file
- * methods and thumbnail generation as for any other
- * Kirby files. Pass a relative path to the Asset
- * object to create the asset.
+ * methods as for any other Kirby files. Pass a
+ * relative path to the class to create the asset.
  *
- * @package   Kirby Cms
+ * @package   Kirby Filesystem
  * @author    Bastian Allgeier <bastian@getkirby.com>
  * @link      https://getkirby.com
  * @copyright Bastian Allgeier GmbH
  * @license   https://getkirby.com/license
  */
 class Asset
 {
-    use FileFoundation;
-    use FileModifications;
```

Comment:
Oh, one thing I just noticed: Why was `FileModifications` removed?

Reply:
Good catch. I think reviewing all the code, I was really convinced that `FileModifications` in there was a mistake and wouldn't actually work. Now looking at it a week later, I think it does work and I was mistaken. We should add it back in then.

Reply:
What I think is also a bit weird is that the `Asset` class gets an `asset()` method from `IsFile`. But as that's always been the case, I think we should leave it in to avoid even more breaking changes.

Reply:
Yea, I have also moved in circles with the naming here. You quickly end up with having a `file()` method in `File` etc.

Reply:
@lukasbestle have added it again